vagga.yaml

```diff
@@ -24,3 +69,18 @@ commands:
     description: Shortcut to manage.py
     container: django
     run: [python3, manage.py]
+  run-postgres: !Supervise
+    description: Start the django development server using Postgres database
+    children:
+      app: !Command
+        container: django
+        environ:
+          DATABASE_URL: postgresql://vagga:vagga@127.0.0.1:5433/test
+        run: |
+            python3 manage.py migrate
+            python3 manage.py runserver
+      db: !Command
+        container: postgres
+        user-id: 200
+        group-id: 200
+        run: exec $PG_BIN/postgres -F --port=$PG_PORT
```

Comment:
This also come with error 
`ERROR:vagga::wrapper: Error executing run-postgres: Command exec not found in "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
---------- Process db:8849 exited with code 124. Shutting down -----------`